src/element.ts

```diff
@@ -37,7 +37,7 @@ export interface ElementStyle {
  */
 export abstract class Element {
   protected static ID: number = 1000;
-  protected context?: RenderContext;
+  public context?: RenderContext;
```

Comment:
Is there a `context` reference somewhere in the code? (Can it be replaced with `getContext/setContext`?

Reply:
I have left a comment below in note.js L505 where the reference is.

Reply:
you are right